mmv1/products/metastore/api.yaml

```diff
@@ -241,7 +265,22 @@ objects:
         name: 'uid'
         output: true
         description: |
-          The globally unique resource identifier of the metastore service.          
+          The globally unique resource identifier of the metastore service.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'metadataIntegration'
+        min_version: beta
+        description: |
+          The setting that defines how metastore metadata should be integrated with external services and systems.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'dataCatalogConfig'
+            description: |
+              The integration config for the Data Catalog service.
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: 'enabled'
+                description: |
+                  Defines whether the metastore metadata should be synced to Data Catalog. The default value is to disable syncing metastore metadata to Data Catalog.      
```

Comment:
Can we make both `dataCatalogConfig` and `enabled` required to prevent users from sending empty block?

Reply:
added. sorry for delay